web/core/components/cycles/analytics-sidebar/sidebar-header.tsx

```diff
@@ -156,7 +156,7 @@ export const CycleSidebarHeader: FC<Props> = observer((props) => {
       end_date: renderFormattedPayloadDate(endDate),
     };
 
-    if (cycleDetails?.start_date && cycleDetails.end_date)
+    if (cycleDetails && cycleDetails.start_date && cycleDetails.end_date)
```

Comment:
Any reason for this change? both looks same

Reply:
I've reverted the change.